README.md

```diff
@@ -1,85 +1,94 @@
 # Elide
```

Comment:
I think you meant to update the README.MD file under translations/zh instead of this one.